connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -223,7 +232,7 @@ export async function notionUpsertPageActivity(
 
   if (alreadySeenInRun) {
     localLogger.info("Skipping page already seen in this run");
-    return;
+    return { document: notionPage, createdOrMoved: false };
```

Comment:
Same here, I think `document` is kind of confusing, especially as only `pages` end up being documents